test/uk/gov/hmrc/customs/emailfrontend/views/LayoutSpec.scala

```diff
@@ -47,13 +47,13 @@ class LayoutSpec extends SpecBase {
         shouldContainCorrectBanners(layoutView)
       }
 
-      "there is no value for title and back link" in new Setup {
+      "there is no value for title and no back link" in new Setup {
 
         val layoutView: Document = Jsoup.parse(app.injector.instanceOf[Layout].apply()(content).body)
 
         shouldContainCorrectTitle(layoutView)
         shouldContainCorrectServiceUrls(layoutView.html())
-        shouldContainCorrectBackLink(layoutView)
+        layoutView.getElementsByClass("govuk-back-link").text() mustBe emptyString
```

Comment:
Better to create a private method like below to maintain the existing tests' readability consistency and move this into that method

shouldNotContainBackLink